src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: #000;
+  --hover-color: #00acdc;
+  --header-background-color: #fff;
+}
+
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
```

Comment:
The class name '.header' could be more descriptive of the content rather than the style or tag name. Consider using a class name that represents the role or content of the element.